docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx

```diff
@@ -80,7 +80,10 @@ See our [FedRAMP Compliance for Infrastructure Access](../../access-controls/com
 
 ## Access Requests
 
-With Teleport we've introduced the ability for users to request additional roles. The Access Request API makes it easy to dynamically approve or deny these requests.
+With Teleport, users can request access roles or to a specific set of resources.
+Requests can be published to tools like Slack, Jira, and PagerDuty for easy integration with your organization's workflows.
+requesting specific resources such as a production server to allow fine-grain access for a targeted task.  The web interface
```

Comment:
> requesting specific resources such as a production server to allow fine-grain access for a targeted task.  

Looks like the first half of this sentence was cut off; there's no subject. Also, extra spacing at the end.

Reply:
Please see updates